esis/science/papers/instrument/sections/introduction.py

```diff
@@ -10,12 +10,27 @@ def section() -> pylatex.Section:
     result.escape = False
     result.append(
         r"""
-The solar \TR\ and corona, as viewed from space in its characteristic short wavelengths (\FUV, \EUV, and soft X-ray), 
-is a three-dimensional scene evolving in time:  $I(x, y, \lambda, t)$.
-Here, the helioprojective cartesian coordinates, $x$ and $y$ \citep{Thompson2006}, and the wavelength axis, $\lambda$, 
-comprise the three dimensions of the scene, while $t$ represents the temporal axis.
-An ideal instrument would capture a spatial/spectral data cube, at a rapid temporal cadence, however, practical 
-limitations lead us to accept various compromises of the sampling rate along each of these four dimensions.
+The light emitted by the solar \TR\ and corona varies significantly as a function
+of position, wavelength, and time.
+When viewed from Earth, the spectral radiance from the Sun can be written as: $I(x, y, \lambda, t)$, 
+where $x$ and $y$ are the helioprojective Cartesian coordinates \citep{Thompson2006}, 
+$\lambda$ is wavelength, and $t$ is time.
+The ideal solar imaging spectrograph would capture $I(x, y, \lambda, t)$ with high resolution in $x$, $y$, $\lambda$, 
+and $t$ \textit{and} over a wide \FOV\, wavelength range, and time period.
+Of course, the temporal dimension is privileged, so we often reduce the problem to capturing a 3D spatial/spectral
+cube at a particular time $t_0$: $I(x, y, \lambda, t_0)$.
+Since we use 2D detectors, this means that we must find a way to flatten the 3D cube into two dimensions
```

Comment:
...we use 2D detectors to capture an exposure at some time $t$...
Instead of "the 3D cube...without losing information", how about "the remaining three dimensions $x$, $y$, $\lambda$. This will probably entail some loss of information, which we must manage to meet our scientific goals."